tool/tbot/spiffe.go

```diff
@@ -21,22 +21,48 @@ package main
 import (
 	"context"
 	"fmt"
+	"log/slog"
 	"time"
 
 	"github.com/gravitational/trace"
 	"github.com/spiffe/go-spiffe/v2/svid/jwtsvid"
 	"github.com/spiffe/go-spiffe/v2/workloadapi"
-
-	"github.com/gravitational/teleport/lib/utils"
 )
 
+// TODO(tross/noah): Remove once go-spiff has a slog<->workloadapi.Logger adapter.
+// https://github.com/spiffe/go-spiffe/issues/281
+type logger struct {
+	l *slog.Logger
+}
+
+func (l logger) Debugf(format string, args ...interface{}) {
+	//nolint:sloglint // msg cannot be constant
+	l.l.DebugContext(context.Background(), fmt.Sprintf(format, args...))
+}
+
+func (l logger) Infof(format string, args ...interface{}) {
+	//nolint:sloglint // msg cannot be constant
+	l.l.InfoContext(context.Background(), fmt.Sprintf(format, args...))
+}
+
+func (l logger) Warnf(format string, args ...interface{}) {
+	//nolint:sloglint // msg cannot be constant
+	l.l.WarnContext(context.Background(), fmt.Sprintf(format, args...))
+}
+
+func (l logger) Errorf(format string, args ...interface{}) {
+	//nolint:sloglint // msg cannot be constant
+	l.l.ErrorContext(context.Background(), fmt.Sprintf(format, args...))
+}
+
 func onSPIFFEInspect(ctx context.Context, path string) error {
 	log.InfoContext(ctx, "Inspecting SPIFFE Workload API Endpoint", "path", path)
 
 	source, err := workloadapi.New(
 		ctx,
 		// TODO(noah): Upstream PR to add slog<->workloadapi.Logger adapter.
-		workloadapi.WithLogger(utils.NewLogger()),
+		// https://github.com/spiffe/go-spiffe/issues/281
+		workloadapi.WithLogger(logger{l: slog.Default()}),
```

Comment:
Maybe use a specialized component for this logger, or some other distinctive With key/value?

Reply:
Happy to keep it as-is if you'd rather avoid other changes here.

Reply:
I'm happy with this for now. I'll revisit all of tbot daemon logs at somepoint